client/src/styles/EmojiPicker.scss

```diff
@@ -0,0 +1,36 @@
+.emoji-picker {
+  width: 235px;
+  background: #fff;
+  box-sizing: border-box;
+  border-radius: 2px;
```

Comment:
Let's use no border-radius here. I think it'll go better with the overall "square" style of Orbit.